nemo/collections/tts/modules/common.py

```diff
@@ -158,13 +159,13 @@ def __init__(
             self.dense = nn.Linear(n_channels, out_dim)
 
     def forward(self, context: Tensor, lens: Tensor) -> Tensor:
-        mask = get_mask_from_lengths(lens, context)
+        mask = get_mask_from_lengths(lens)
         mask = mask.to(dtype=context.dtype).unsqueeze(1)
+        context = context[:, :, : mask.shape[-1]]
```

Comment:
Why are we dropping the masking here?

Reply:
Why, it's not dropping it - it's making context the shape of the mask, not vice versa. Required to match ONNX operation. I actually am already moving that truncation higher up to radtts.py so it would be more intuitive and more efficient.

Reply:
Can you point me to where the masking is done now?

Reply:
Actually, I see that now you reverted this change. Was that intentional?

Reply:
Yes - I have moved slicing up (and out), so mask length is determined by input context, as before.